airflow/utils/dag_processing.py

```diff
@@ -1268,6 +1268,9 @@ def _find_zombies(self, session):
             )
             self._last_zombie_query_time = timezone.utcnow()
             for ti in tis:
+                sti = SimpleTaskInstance(ti)
+                self.log.info("=> Failing job with dag_id %s, task_id %s, and execution date %s",
+                              sti.dag_id, sti.task_id, sti.execution_date)
                 zombies.append(SimpleTaskInstance(ti))
```

Comment:
```suggestion
                zombies.append(sti)
```

No need to create the object twice.

Reply:
Thanks for catching that. I've made the change. :+1: